.github/workflows/test-suite.yml

```
@@ -2,24 +2,47 @@ name: 'Test Suite'
 
 on:
   workflow_dispatch:
+    inputs:
+      terraform_matrix:
+        description: 'Terraform version matrix (JSON array)'
+        default: '["1.6.x", "1.5.x", "1.4.x", "1.3.x", "1.2.x"]'
+      provider_matrix:
+        description: 'MongoDB Atlas Provider version matrix (JSON array)'
```

```suggestion
        description: 'MongoDB Atlas Provider version matrix for running migration tests (JSON array)'
```
WDYT about making it more explicit? Without too much context you can understand that the provider version used for running tests is defined here.

---

sounds good, suggestion accepted